Tests/BaggageContextTests/BaggageContextTests.swift

```diff
@@ -0,0 +1,21 @@
+import BaggageContext
+import XCTest
+
+final class BaggageContextTests: XCTestCase {
+    func testSubscriptAccess() {
+        let value = 42
+
+        var baggage = BaggageContext()
+        XCTAssertNil(baggage[TestContextKey.self])
+
+        baggage[TestContextKey.self] = value
+        XCTAssertEqual(baggage[TestContextKey], value)
+
+        baggage[TestContextKey.self] = nil
+        XCTAssertNil(baggage[TestContextKey.self])
+    }
+}
```

Comment:
can we also add a test for the pattern we're recommending? Just so people can spot it here as well.

(the extension for `baggage.testContextKey`

Reply:
Yes, makes sense 👍